lib_uart/src/uart_fast_tx.xc

```diff
@@ -16,6 +16,7 @@ void uart_tx_streaming_write_byte(streaming chanend c, uint8_t byte)
 void uart_tx_streaming(out port p, streaming chanend c, int clocks) {
     int t;
     unsigned char b;
+    p <: 1; // idle state for uart
```

Comment:
is this a bug fix? If so we should put in the the changelog.

Reply:
It was part of the reason one of the tests was uncommented, so I guess it is a bugfix